src/app/components/Filters/SelectedFilters.jsx

```diff
@@ -143,32 +145,35 @@ class SelectedFilters extends React.Component {
     }
 
     _mapObject(selectedFilters, (values, key) => {
-      if (_contains(acceptedFilters, key) && values && values.length) {
-        if (key === 'dateAfter' || key === 'dateBefore') {
-          datesToRender.push(_extend(
-            { field: key },
-            {
-              value: values,
-              label: values,
-            },
-          ));
-        } else {
-          if (_isArray(values)) {
-            values.forEach(value =>
-              filtersToRender.push(_extend({ field: key }, value)));
-          } else {
-            filtersToRender.push(_extend(
+      if (key === 'subjectLiteral' || !dropdownOpen) {
```

Comment:
I don't really think you need this check since you have included 'subjectLiteral' in appConfig.

The other thing is I am not sure if we want another 'subjectLiteral' on the top of the filter pop up.

Reply:
I think this is outdated now, we are going to include the subject with the other filters